.github/workflows/label-zero-impact.js

```diff
@@ -0,0 +1,68 @@
+const { getLocalConfigs } = require('./helpers.js');
+const zeroImpactDirs = ['.github', '.kodiak', '.vscode', '.test', 'libs/mep'];
```

Comment:
What about changes to CODEOWNERS or similar files in the root? I think on a PR like this https://github.com/adobecom/milo/pull/2194, the `zero-impact` label would be removed.

Reply:
Added a few more files & we can always enhance the list as we see the need.
If we wrongfully remove the `zero-impact` label, your PR would still get merged regularly with the next batch.

But for developing workflows / making repo adjustments / mep / unit tests, this will come in handy to reduce the number of open PRs.